### PR TITLE
[PATCH] [Outreachy] psuh: outreachy applicant myfirstcontribution tutorial

### PR DESCRIPTION
**Pull request by Georgecanfly**
/allow <Georgecanfly>

This is a toy function called psuh which surprises a user when they have a typo with a pony saying, "Um, Hello". This also includes parameters, the current branch, displays the top of origin/master, parse options, a documentation man page and a test. It is from the MyFirstContribution tutorial and my first real Pull Request/Patch Review. 

/preview

### CODE REVIEW

File: git.c [hunk 1]
```diff
@@ -546,6 +546,7 @@ static struct cmd_struct commands[] = {
 	{ "pickaxe", cmd_blame, RUN_SETUP },
 	{ "prune", cmd_prune, RUN_SETUP },
 	{ "prune-packed", cmd_prune_packed, RUN_SETUP },
+        { "psuh", cmd_psuh, RUN_SETUP },
```
Comment (nasamuffin) [inline git.c]: The Git project uses tabs, not spaces - that's why your whitespace looks weird here. I suggest you set your editor accordingly :)

**Comment by nasamuffin** [comment]
Your patchset mostly looks good! Definitely don't send this to the mailing list, as you know :)

File: git.c (hunk 1 above)
Comment (Georgecanfly) [reply git.c]: All right! Thanks. I saw that and went back to open the file and it looked normal. Makes more sense now. Thanks @nasamuffin

Comment (Georgecanfly) [reply git.c]: i did git push -f Georgecanfly psuh to send updated branch in my fork. Now this says Outdated. Not sure if it's correct.

**Comment by Georgecanfly** [comment]
This is a toy function called psuh which surprises a user when they have a typo with a pony saying, "Um, Hello". This also includes parameters, the current branch, displays the top of origin/master, parse options, a documentation man page and a test. It is from the MyFirstContribution tutorial and my first real Pull Request/Patch Review.

NEW: I went ahead and committed my branch again fixing whitespace errors found in my git.c file.  

/preview

**Comment by dscho** [comment]
> /preview

You will want to send this as a separate comment. It will take the PR description and send it as cover letter.

**Comment by dscho** [comment]
@Georgecanfly did it work all right?

(And: do you want to close this PR now?)

**Pull request closed without merging**